index.d.ts

```diff
@@ -99,11 +99,9 @@ declare namespace chalk {
 		@param color - Hexadecimal value representing the desired color.
 
 		@example
-		```
```

Comment:
These are unnecessary as `@example` already creates a code block and using these causes the code example to be indented unnecessarily in **VS Code’s** IntelliSense popout.

Reply:
This conforms with @sindresorhus's [TS styleguide](https://github.com/sindresorhus/typescript-definition-style-guide).

Reply:
Well, then @sindresorhus’s style guide should be amended.

Reply:
Without them VS Code's popout doesn't preserve tab-indentation. I'm also not really interested in working around VS Code issues. Open a VS Code issue about this instead. I'm not going to change the style guide about this.